index.bs

```diff
@@ -302,6 +302,12 @@ This specification defines a [=policy-controlled feature=] identified by the str
 
 # Structures # {#structures}
 
+<h3 dfn-type=dfn>Attribution mode</h3>
+
+An attribution mode is a [=tuple=] consisting of a mode and a random trigger data. The mode is
```

Comment:
This works for views because a report is completely described by its trigger data. This is OK for now but it's worth thinking about a future where we need more parameters like the report time.

One thing to consider is having a tuple of (mode, {set of attribution reports}) where the set of attribution reports are all of the fake reports to be generated by a particular source.

Reply:
I think we can add that complexity when it's needed.

Reply:
Yep totally fine adding it later, just wanted to highlight it.